README.md

```diff
@@ -2,7 +2,7 @@
 
 [![NPM version][npm-image]][npm-url] [![Downloads][npmcharts-image]][npmcharts-url] [![Gitter][gitter-badge-image]][gitter-badge-url] [![Known Vulnerabilities][snyk-image]][snyk-url] [![Package Health][health-image]][health-url]
 
-[![Generator Build Status][github-actions-generator-image]][github-actions-url] [![Angular Build Status][github-actions-angular-image]][github-actions-url] [![React Build Status][github-actions-react-image]][github-actions-url] [![Vue Build Status][github-actions-vue-image]][github-actions-url] [![Revved up by Gradle Enterprise](https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.jhipster.tech/scans)
+[![Generator Build Status][github-actions-generator-image]][github-actions-url] [![Angular Build Status][github-actions-angular-image]][github-actions-url] [![React Build Status][github-actions-react-image]][github-actions-url] [![Vue Build Status][github-actions-vue-image]][github-actions-url] [![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.jhipster.tech/scans)
```

Comment:
The website at https://ge.jhipster.tech/scans still says Gradle Enterprise.

Reply:
Thanks for mentioning. That will be updated in the coming months. Product renames are a lot more difficult than badge renames :)

Reply:
Do you have a link to the product rename announcement?

Reply:
@bdemers Can you confirm this is legit?

Reply:
Confirmed!

TL;DR, it was hard to explain to folks that Gradle Enterprise was not an enterprise distribution of Gradle Build Tool 😆

Reply:
> Do you have a link to the product rename announcement?

https://gradle.com/press-media/gradle-enterprise-is-now-develocity/